README.md

```diff
@@ -6,7 +6,23 @@ Release automation for pandas.
   - [  ] `TAG` in `Makefile`
   - [  ] `GH_USERNAME` in `Makefile`
 
-If running for the first time be sure to initialize repos
+## **If running for the first time:**
+
+- Create a conda enviroment based on environment.yml
+```sh
+conda env create -f environment.yml
```

Comment:
You don't need the `-f environment.yml` if the name of the file is `environment.yml`, since it's conda's default. Feel free to leave it if you prefer to be explicit.

Reply:
I didn't know that was the default, good to know.